include/eve/function/sincosd.hpp

```diff
@@ -14,6 +14,6 @@ namespace eve
   EVE_MAKE_CALLABLE(sincosd_, sincosd);
 }
 
-#include <eve/module/real/core/function/scalar/sincosd.hpp>
-//#include <eve/module/real/core/function/regular/simd/sincosd.hpp>
+#include <eve/module/core/function/scalar/sincosd.hpp>
+//#include <eve/module/core/regular/impl/simd/sincosd.hpp>
```

Comment:
why the comment ?

Reply:
the file as to be suppressed  sindcosd is the good one
I'll do it